@here/harp-omv-datasource/lib/OmvDecoder.ts

```diff
@@ -695,11 +697,32 @@ export class OmvTileDecoder extends ThemedTileDecoder {
 
     private createFeatureModifier(
         filterDescription: OmvFeatureFilterDescription,
-        featureModifierId?: FeatureModifierId
+        featureModifierId?: FeatureModifierId,
+        respectedPov?: string | null
     ): OmvFeatureModifier {
         if (featureModifierId === FeatureModifierId.tomTom) {
+            assert(
+                respectedPov === undefined,
+                "TomTom does not support different political views!"
+            );
             return new OmvTomTomFeatureModifier(filterDescription);
+        } else if (
+            respectedPov !== undefined &&
+            respectedPov !== null &&
+            respectedPov.length === 2
+        ) {
+            return new OmvPointOfViewFeatureModifier(filterDescription, [respectedPov]);
         } else {
+            assert(
```

Comment:
This validation belongs to constructor of `OmvPointOfViewFeatureModifier`

Reply:
It depends on the solution chosen above, if we go with empty string for default point of view, I would not even create OmvPointOfViewFeatureModifier(), which gives overhead, but instead just use OmvGenericFeatureModifier.
Let's analyze it after choosing approach from above comments.